torch/_inductor/codegen/cpp.py

```diff
@@ -1410,6 +1412,10 @@ def codegen_loops(self, code, worksharing):
         loop_nest = LoopNestWithSplit.build(self)
         self.codegen_loops_impl(loop_nest, code, worksharing)
 
+    @property
+    def assert_function(self):
+        return "TORCH_CHECK"
```

Comment:
You will probably need a vectorized version as well which checks all elements of the vector are true.

Reply:
Peter, do you happen to know a test case or an example that requires a vectorized version of `TORCH_CHECK`?

Reply:
See https://github.com/pytorch/pytorch/pull/108690#discussion_r1332273331